sympy/printing/tests/test_str.py

```diff
@@ -660,14 +660,14 @@ def test_settings():
 def test_RandomDomain():
     from sympy.stats import Normal, Die, Exponential, pspace, where
     X = Normal('x1', 0, 1)
-    assert str(where(X > 0)) == "Domain: And(0 < x1, x1 < oo)"
+    assert str(where(X > 0)) == "Domain: (0 < x1 & x1 < oo)"
 
     D = Die('d1', 6)
-    assert str(where(D > 4)) == "Domain: Or(Eq(d1, 5), Eq(d1, 6))"
+    assert str(where(D > 4)) == "Domain: (Eq(d1, 5) | Eq(d1, 6))"
 
     A = Exponential('a', 1)
     B = Exponential('b', 1)
-    assert str(pspace(Tuple(A, B)).domain) == "Domain: And(0 <= a, 0 <= b, a < oo, b < oo)"
+    assert str(pspace(Tuple(A, B)).domain) == "Domain: (0 <= a & 0 <= b & a < oo & b < oo)"
```

Comment:
> Do we really need parentheses around the inequalities in that case? Don't relational operators have precedence over logical And? (at least from a programming language perspective)

No they don't. That's why parentheses are so important.

```
In [71]: type(x < y & z)
Out[71]: sympy.core.relational.StrictLessThan

In [72]: x < (y & z)
Out[72]: x < y ∧ z

In [73]: (x < y) & z
Out[73]: z ∧ x < y

In [74]: (x < y & z).args
Out[74]: (x, y ∧ z)
```

See also https://docs.python.org/3/reference/expressions.html#operator-precedence